crates/solidity/testing/sourcify/src/run.rs

```diff
@@ -211,14 +243,175 @@ fn run_bindings_check(
                     version = contract.version,
                 ));
 
-                test_outcome = TestOutcome::Failed;
+                test_outcome = TestOutcome::Unresolved;
             }
         }
     }
 
     test_outcome
 }
 
+fn build_new_binder_output(compilation_unit: CompilationUnit) -> Output {
```

Comment:
I wonder if the list of passes should be defined in `slang_solidity::backend` instead?
Why the duplication here?